tools/src/gt_mpi_gather.cc

```diff
@@ -354,10 +354,23 @@ void scan_and_produce_Broad_GVCF(const VariantQueryProcessor& qp, const VariantQ
   delete op_ptr;
 }
 
-void iterate_columnar_gvcf(const VariantQueryProcessor& qp, const VariantQueryConfig& query_config, int command_idx) {
-  ProfilerOperator counter;
-  qp.iterate_over_gvcf_entries(qp.get_array_descriptor(), query_config, counter, true);
-  std::cerr << "Counter "<<counter.get_value() << "\n";
+void iterate_columnar_gvcf(const VariantQueryProcessor& qp, const VariantQueryConfig& query_config, int command_idx,
```

Comment:
Need to add gt_mpi_gather tests to either `run.py` or `test_tools.sh` to sanity test iterate_columnar_gvcf and columnar iterators in general.

Reply:
Not yet, the unit tests in test_columnar_gvcf_iterator handle all the checking we need currently.
Once the full VCF functionality is implemented, we can add one extra command for columnar iterator wherever produce-Broad-GVCF is invoked